local-development/the-run-script.md

```diff
@@ -0,0 +1,43 @@
+# Running our projects with the ./run script
+
+Each of our projects should conform to exactly the same local developmet interface,
+which should be documented in the project's `README.md`. This should make it simple
+for anyone to run any of our projects.
+
+This interface is the `./run` script. Each project's `./run` script should take the same set of commands.
+The script will (at the time of writing) use [Docker](https://www.docker.com/community-edition) containers to spin up each of the required services
```

Comment:
You should just leave it out - all best practices are subject to change in the future. If and when they do, we'll update this doc. 👍🏻